docs/user/reference/environment-variables.rst

```diff
@@ -56,6 +56,31 @@ All :doc:`build processes </builds>` have the following environment variables au
     :Example: ``https://docs.readthedocs.io/ja/stable/``
     :Example: ``https://example--17.org.readthedocs.build/fr/17/``
 
+.. envvar:: READTHEDOCS_REPOSITORY_CLONE_URL
+
+    URL for the remote source repository, from which the documentation is cloned.
+    It could be HTTPS, SSH or any other URL scheme supported by Git.
+    This is the same URL defined in your Project's :term:`dashboard` in :menuselection:`Admin --> Settings --> Repository URL`.
+
+    :Example: ``https://github.com/readthedocs/readthedocs.org``
+    :Example: ``git@github.com:readthedocs/readthedocs.org.git``
+
+.. envvar:: READTHEDOCS_REPOSITORY_IDENTIFIER
+
+    Contains the Git identifier that was *checked out* from the remote repository URL.
+    Possible values are either a branch or tag name.
+
+    :Example: ``v1.x``
+    :Example: ``bugfix/docs-typo``
+    :Example: ``feature/signup``
+    :Example: ``update-readme``
+
+.. envvar:: READTHEDOCS_REPOSITORY_IDENTIFIER_HASH
```

Comment:
I don't love how long this name is, but it seems reasonably explicit. 

I think IDENTIFIER and HASH are kind of the same thing, but I'm not sure if git has a canonical name for this. I couldn't quickly find it in the docs.